pages/index.js

```diff
@@ -1,3 +1,36 @@
+import fetcher from "../helpers/fetcher";
+import useSWR from "swr";
+import styled from "styled-components";
+import Head from "next/head";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data, error } = useSWR("api/facilities", fetcher);
```

Comment:
```suggestion
  const { data, error } = useSWR("/api/facilities", fetcher);
```

it's always a good idea to use absolute paths for endpoints